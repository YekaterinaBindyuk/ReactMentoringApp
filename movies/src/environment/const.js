export const BASE_IMG_CLASSNAME = 'img';
export const MOVIE_LIST_ITEM_COVER = 'small';
export const MOVIE_COVER = 'large';
export const MAIN_LOGO = 'http://localhost:8080/src/images/logo.png';
export const SEARCH_BUTTON = 'Search';
export const TITLE_TYPE_BUTTON = 'title';
export const GENRE_TYPE_BUTTON = 'genre';
export const WARNING = 'Something went wrong';
export const RATING_TYPE_BUTTON = 'rating';
export const RELEASE_TYPE_BUTTON = 'release date';
export const SORTING_TEXT = 'Sort by: ';
export const SEARCH_TEXT = 'Search by: ';
export const NO_FILMS_WARNING = 'No films found';
export const MOVIES_COUNT_TEXT = ' films found:';
export const RECOMMENDED_MOVIES = 'Recommended movies';
export const MAIN_PAGE = '/';
export const REST_URL = 'https://reactjs-cdp.herokuapp.com/';
export const MOVIES_URL = REST_URL + 'movies';
