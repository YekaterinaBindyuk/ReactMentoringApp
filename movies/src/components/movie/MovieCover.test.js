import React from "react";
import renderer from "react-test-renderer";
import { BASE_IMG_CLASSNAME } from "../../environment/const";

it("renders correctly", () => {
  const title = "test title";
  const type = "small";
  const url = "https://image.tmdb.org/t/p/w500/kOVEVeg59E0wsnXmF9nrh6OmWII.jpg";

  const tree = renderer
    .create(
      <img src={url} alt={title} className={`${BASE_IMG_CLASSNAME}_${type}`} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot(title, url, type);
});
