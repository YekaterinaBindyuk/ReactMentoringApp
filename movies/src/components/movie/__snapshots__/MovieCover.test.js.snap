// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: test title 1`] = `
<img
  alt="test title"
  className="img_small"
  src="https://image.tmdb.org/t/p/w500/kOVEVeg59E0wsnXmF9nrh6OmWII.jpg"
/>
`;
